mne/viz/_brain/_brain.py

```diff
@@ -1270,10 +1272,8 @@ def save_movie(filename, weakself=weakself):
         )
 
     def _shift_time(self, op):
```

Comment:
since we're touching these internal methods anyway, do you think we could change the `op` parameter name? Since we usually import `os.path` as `op` too, I got a little confused when I looked at the code in the body of the method for the first time :)